### DISCUSSION TITLE
Can't install pix2tex[gui]

### DISCUSSION THREAD

**Discussion by XaviereKU** [lukas-blecher/LaTeX-OCR#211] [Q&A] [answered]
I'm using M1 Macbook Air and try to install pix2tex[gui](actually all)
but when I try
pip install pix2tex[gui]
in terminal, it says
zsh : no matches found: pix2tex[gui]

Still, I can install pix2tex, and it runs well.
But I want to install other packages and use it through importing it in my python project.

What should I do?

**Comment by lukas-blecher** [comment, marked as answer]
It's a zsh thing.
Try
```
pip install "pix2tex[gui]"
```
